spec/requests/bids_spec.rb

```diff
@@ -53,6 +54,10 @@
       it 'returns a 404 status code' do
         expect(status).to eq(404)
       end
+
+      it 'should not create a bid' do
+        expect(auction.bids.count).to eq(@bid_count)
```

Comment:
since ivars can be reassigned wherever, not sure I am a big fan of using them for rspec expectations.

thoughts on moving the "exercise" into each spec (`post auction_bids_path(auction), params, headers`) and then testing it like this:

```ruby
expect do
  post auction_bids_path(auction), params, headers
end.not to { auction.bids.count }
```